app/views/backend/purchases/_form.html.haml

```diff
@@ -13,6 +13,7 @@
   = f.input :tax_payability, as: :radio_buttons
   = f.referenced_association :responsible, source: :employees, new: { employed: 1 }
   = f.input :description
+  = f.association :creator, input_html: { value: current_user.id }, as: :hidden
```

Comment:
Risky. If we need this, it needs to be moved up into the controller.
Also: don't timestamps already handle this?